test/E2E/TestFunctionApp/extensions.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
```

Comment:
Do we need this `WarningsAsErrors`?

Reply:
Replied on another PR: this code is auto-generated, so I think we should leave it alone. If you disagree, please let me know, otherwise I'm merging it as is.